tuftool/Cargo.toml

```
@@ -42,4 +44,6 @@ walkdir = "2"
 
 [dev-dependencies]
 assert_cmd = "2"
+futures = "0.3.28"
+futures-core = "0.3.28"
```

Can you reduce the specificity of the dependency versions? We have seen conflicts with other downstream dependencies at times when they require a "no-higher-than X" version.

```suggestion
futures = "0.3"
futures-core = "0.3"
```